web/annotation_editor_layer_builder.css

```diff
@@ -332,3 +366,77 @@
     }
   }
 }
+
+.altText {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: auto;
+  height: 24px;
+  min-width: 88px;
+  position: absolute;
+  bottom: 8px;
+  left: 8px;
```

Comment:
Here, and below, what happens in RTL mode when we use `left` here?
Should we perhaps use `inset-inline-start` instead to be direction aware?

Reply:
Good question, the design (which have validated by a bunch of persons) shows that the button is in the bottom left, so I'd we should keep it like this for the moment, but I'll raise the question to UX/UI.